sources/Outputs/src/ParGenerator.cpp

```diff
@@ -254,5 +262,15 @@ ParGenerator::updatePropParameters(boost::shared_ptr<parameters::ParametersSet>
   set->addReference(helper::buildReference("generator_QPercent", "qMax_pu", "DOUBLE"));
 }
 
+void
+ParGenerator::updateTransfoParameters(const algo::GeneratorDefinition& generator, boost::shared_ptr<parameters::ParametersSet> set) {
+  // TODO(rosiereflo) We assume here that QNom = QMax which might not be always true ...
+  set->addReference(helper::buildReference("generator_QNomAlt", "qMax", "DOUBLE"));
+  set->addParameter(helper::buildParameter("generator_SNom", sqrt(generator.pmax * generator.pmax + generator.qmax * generator.qmax)));
```

Comment:
We should keep in mind that the qmax must be the largest qmax of the diagram and not the one at p

Reply:
there is a todo above that mentions that

Reply:
Yes, it was just a comment :)